tree-sitter-stack-graphs/src/cli/init.rs

```diff
@@ -103,154 +94,175 @@ impl InitArgs {
+            .validate_with(regex_validator(&VALID_CRATE_NAME))
             .interact_text()?;
-        println!();
 
         printdoc! {r#"
 
-            Give the NPM package name for this project. Must be a valid scoped or unscoped
-            package name.
+            Give the crate name for this project. May only contain letters, numbers, dashes,
+            and underscores.
             "#
         };
-        let project_npm_name: String = Input::new()
-            .with_prompt("Project NPM package name")
-            .with_initial_text("tree-sitter-stack-graphs-".to_string() + &language_id)
-            .validate_with(regex_validator(&VALID_NPM_PKG))
+        let default_crate_name = "tree-sitter-stack-graphs-".to_string() + &self.language_id;
+        self.crate_name = Input::new()
+            .with_prompt("Package name")
+            .with_initial_text(if self.crate_name.is_empty() {
+                &default_crate_name
+            } else {
+                &self.crate_name
+            })
+            .validate_with(regex_validator(&VALID_CRATE_NAME))
             .interact_text()?;
-        println!();
 
         printdoc! {r#"
 
-            Give the NPM package version for this project.
+            Give the crate version for this project. Must be in MAJOR.MINOR.PATCH format.
             "#
         };
-        let project_npm_version: String = Input::new()
-            .with_prompt("Project NPM package version")
-            .with_initial_text("0.1.0")
-            .validate_with(regex_validator(&VALID_NPM_VERSION))
+        self.crate_version = Input::new()
+            .with_prompt("Package version")
+            .with_initial_text(if self.crate_version.is_empty() {
+                "0.1.0"
+            } else {
+                &self.crate_version
+            })
+            .validate_with(regex_validator(&VALID_CRATE_VERSION))
             .interact_text()?;
-        println!();
 
         printdoc! {r#"
 
             Give the project author in the format NAME <EMAIL>. Leave empty to omit.
             "#
         };
-        let project_author: String = Input::new()
+        self.author = Input::new()
             .with_prompt("Author")
+            .with_initial_text(&self.author)
             .allow_empty(true)
             .interact_text()?;
-        println!();
 
         printdoc! {r#"
 
             Give the project license as an SPDX expression. Leave empty to omit.
             "#
         };
-        let project_license: String = Input::new()
+        self.license = Input::new()
             .with_prompt("License")
+            .with_initial_text(&self.license)
             .allow_empty(true)
             .interact_text()?;
-        println!();
 
         printdoc! {r#"
 
-            Give the NPM package name for the Tree-sitter grammar that is to be used for
-            parsing.
+            Give the crate name for the Tree-sitter grammar that is to be used for
```

Comment:
s/Tree-sitter/Tree-Sitter

Reply:
Lowercase "s" is what the website https://tree-sitter.github.io/tree-sitter/ also uses.

Reply:
Oh no!  I've flipped this in my mind at some point and thought it was "Tree-Sitter" 🤦 ,  thanks for clarifying.